public/index.html

```diff
@@ -1,284 +1,455 @@
-            <img class="size-5" src="./images/o.svg" alt="Kółko">
-            <img class="size-5" src="./images/o.svg" alt="Kółko">
-            <img class="size-5" src="./images/o.svg" alt="Kółko">
-            <img class="size-5" src="./images/o.svg" alt="Kółko">
-            <img class="size-5" src="./images/o.svg" alt="Kółko">
-          </div>
+    </section>
+    <section class="w-full">
+      <div
+        class="relative mx-auto flex max-w-7xl flex-col items-start px-4 pt-[74px] text-white md:pt-36 lg:px-8"
+      >
+        <div class="absolute right-10 top-0 hidden lg:right-40 lg:block 2xl:right-0">
+          <img class="mt-12 size-24" src="./images/x.svg" alt="Krzyżyk" />
         </div>
-
-        <div id="event" class="-mt-16 px-2 sm:pl-0 lg:w-full text-center lg:text-left leading-none text-5xl sm:text-7xl lg:text-9xl font-bold">Nowoczesne technologie</div>
-        <div class="mt-8 text-2xl sm:text-4xl flex items-center">
-          <span class="flex-grow px-2">Inteligentne rozwiązania, lokalne podejście</span>
-          <div class="ml-20 bg-[#01FF6B] h-3 w-20"></div>
+        <div class="absolute right-0 top-10 hidden h-24 w-24 lg:block 2xl:right-10">
+          <img class="mt-36 size-12 2xl:ml-40" src="./images/x-white.svg" alt="Krzyżyk" />
         </div>
-        <div x-show="active" class="hidden flex-shrink w-max-2xl bg-black bg-opacity-75 mt-20 px-4 py-6 rounded-2xl sm:flex divide-x divide-slate-100/25">
-          <div class="flex px-4">
-            <div class="text-7xl font-bold text-[#01FF6B]" x-text="counters.days"></div>
-            <div class="rotate-180 text-sm py-2">
-              <div class="rotate-90 px-1">dni</div>
-            </div>
-          </div>
-          <div class="flex px-4">
-            <div class="text-7xl font-bold" x-text="counters.hours"></div>
-            <div class="rotate-180 text-sm py-4">
-              <div class="rotate-90 px-1">godzin</div>
-            </div>
-          </div>
-          <div class="flex px-4">
-            <div class="text-7xl font-bold" x-text="counters.minutes"></div>
-            <div class="rotate-180 text-sm py-4">
-              <div class="rotate-90 px-1">minut</div>
+        <div
+          id="event"
+          class="px-4 text-left text-5xl font-bold leading-none sm:pl-0 sm:text-7xl lg:w-full lg:text-9xl"
+        >
+          Nowoczesne technologie
+        </div>
+        <div class="flex items-center pt-6 text-2xl md:pt-9 md:text-[40px]">
+          <span class="flex-grow px-4 md:px-2">Inteligentne rozwiązania, lokalne podejście</span>
+        </div>
+        <template x-if="meetup">
+          <div
+            x-show="active"
+            class="mt-8 flex h-[35px] flex-col pl-3 md:mt-12 md:flex-row md:divide-x md:divide-slate-100/25"
+          >
+            <div class="flex flex-row items-center pl-2 md:pr-8 lg:pl-0">
+              <image src="./images/icons/calendar.svg" class="size-6 self-center lg:size-4"></image>
```

Comment:
Add alt tag here and in a few other images

Reply:
Added missing alt tags.